pandas/core/frame.py

```diff
@@ -6535,7 +6534,7 @@ def dropna(
     @overload
     def drop_duplicates(
         self,
-        subset: Hashable | Sequence[Hashable] | AbstractSet | None = ...,
+        subset: Hashable | Iterable[Hashable] | set | None = ...,
```

Comment:
No need for `set` anymore now that you've changed `Sequence` to `Iterable`, right?

Reply:
That's true. Updated.